src/components/checkoutlogin.jsx

```diff
@@ -0,0 +1,106 @@
+import React, {Component} from 'react';
+import {Form, Button} from 'react-bootstrap';
+
+//Regex to validate the email address.
+const emailRegex = RegExp(
```

Comment:
❗ there's no need to wrap this regular expression with a `RegExp` call
`/eh/` is a regular expression already